lineapy/plugins/pipeline_writers.py

```diff
@@ -105,6 +110,42 @@ def _write_requirements(self) -> None:
         file.write_text(lib_names_text)
         logger.info(f"Generated requirements file: {file}")
 
+    def _write_module_test(self) -> None:
+        """
+        Write out test scaffolding for refactored code in module file.
+        """
+        # Format components to be passed into file template
+        module_name = f"{self.pipeline_name}_module"
+        test_class_name = f"Test{self.pipeline_name.title().replace('_', '')}"
+        function_metadata_list = [
+            {
+                "function_name": f"get_{node_collection.safename}",
+                "function_arg_names": sorted(
+                    [v for v in node_collection.input_variables]
+                ),
+                "test_case_name": f"test_{node_collection.safename}",
```

Comment:
this name looks incorrect. if node_collection.safename is PSY, the function name should be get_PSY and the test name to test that function should be test_get_PSY (not test_PSY). I feel you dont need a special test_case_name here.

Reply:
Done